dll/apisets/api-ms-win-downlevel-shell32-l1-1-0.spec

```diff
@@ -1,6 +1,6 @@
 
 # This file is autogenerated by update.py
 
-@ stdcall CommandLineToArgvW() shell32.CommandLineToArgvW
-@ stub GetCurrentProcessExplicitAppUserModelID
-@ stub SetCurrentProcessExplicitAppUserModelID
+@ stdcall -version=0xA00+ -arch=win32 CommandLineToArgvW() shell32.CommandLineToArgvW
```

Comment:
CommandLineToArgvW is present in windows 2000 already...

Reply:
Yes, the actual function is there. I am fixing the very apiset entry only.